### ISSUE TITLE
Parallel test execution / sharding

### ISSUE THREAD

**Issue by stefaneidelloth** [theintern/intern#779]
I currently use Karma and I am looking for a possibility to execute my tests (> 2500) in parallel ... to utilize more CPUs ... and save some time. However, in Karma a parallelization/sharding feature is not yet implemented:
https://github.com/karma-runner/karma/issues/439

Does Intern already provide one?. I guess not...  since I could not find any information about it?

**Comment by stefaneidelloth** [comment]
I just saw in the feature matrix that Intern
"Runs tests in parallel for improved performance"
https://theintern.github.io/

Does that parallel "only" mean multiple types of browsers at the same,
or also several tests for one type of browser at the same time?

**Comment by jason0x43** [comment]
Yes, currently it just refers to running the same tests in different browsers at the same time. It sounds like you’re interested in having a test runner partition a test set and run different groups on different instances of th same browser at the same time?

**Comment by stefaneidelloth** [comment]
>It sounds like you’re interested in having a test runner partition a test set and run different groups on >different instances of the same browser at the same time?
Yes, exactly.

**Comment by jason0x43** [comment]
That’s been in the idea bin for a while (#391, #546), but I don’t believe anyone’s spent much time on it yet. It’s certainly something we’d like to support. I also created #780; if we develop a solution for WebDriver tests, the same solution might as well apply to Node.

**Comment by jason0x43** [comment]
Closing this in favor of #546
